src/routes/Stream/Stream.js

```diff
@@ -117,6 +118,10 @@ export default class Stream extends Component {
 
     return (
       <>
+        <Helmet>
+          <title>Hylo: Stream</title>
+          <meta name='description' content='Posts from the stream' />
```

Comment:
I wonder if we want to use the Group's description here? If/for/when these pages are publicly shareable